src/libUtils/Logger.cpp

```diff
@@ -277,7 +278,7 @@ void Logger::LogEpochInfo(const char* msg, const char* function,
     std::time_t curTime = std::chrono::system_clock::to_time_t(
         std::chrono::system_clock::now());
 
-    lock_guard<mutex> guard(m);
+    lock_guard<mutex> guard(m_logMutex);
```

Comment:
It's better to move this lock acquiring before calling `system_clock::now()` because by the time you acquire the lock, `curTime` may already be a bit late (although it's minor difference)

Reply:
ok